lib/riffed/client.ex

```diff
@@ -209,6 +248,8 @@ defmodule Riffed.Client do
             call_thrift(new_client, call_name, args, retry_count + 1)
           err = {:error, _} ->
             {new_client, err}
+          exception = {:exception, exception_record} ->
```

Comment:
I believe so, the code I reference is in `master` and I'm running based of a fork of `elixir-thrift` that points at master, not `0.9.3`